root/report/recordings_no_va_link.tt

```diff
@@ -0,0 +1,15 @@
+[%- WRAPPER 'layout.tt' title=l('Recordings not linked to Various Artists but with
+                                 "Various Artists" as the artist credit') full_width=1 -%]
+
+<h1>[% l('Recordings not linked to Various Artists but with "Various Artists" as the artist credit') %]</h1>
```

Comment:
Maybe "Recordings credited to "Various Artists" but not linked to VA"? Doesn't have to be as descriptive as the text below.